src/main/java/org/hmhb/config/DefaultConfigService.java

```diff
@@ -30,7 +30,12 @@ public DefaultConfigService(
 
         this.publicConfig = new PublicConfig(
                 environment.getProperty("google.oauth.client.id"),
-                environment.getProperty("hmhb.url.prefix")
+                environment.getProperty("hmhb.url.prefix"),
+                environment.getProperty("hmhb.program.startYear.minValue", Integer.class),
```

Comment:
If you don't see it yet, you'll know if you review my tests why I'm uncomfortable with the pattern I have set...